src/Swiper.js

```diff
@@ -540,7 +542,7 @@ export default class SwiperAnimated extends PureComponent {
         rotate = this.pan.x.interpolate({ inputRange: [-400, 0, 400], outputRange: ['-8deg', '0deg', '8deg'] });
         translateY = this.pan.y;
         translateX = this.pan.x;
-        panHandlers = swiper && children.length - 1 !== this.currentIndex[this.guid] ?
+        panHandlers = swiper && children.length - 1 !== this.currentIndex[this.guid] || swipeThroughStack ?
```

Comment:
Nice, could you group the conditions in brackets `()`

Reply:
@chitezh is that better?

Reply:
yeah, that makes more sense now